### PR TITLE
converted os name to lower case

### PR DESCRIPTION
**Pull request by samkeshmiri**
refactored os method to return index of string if it is a mac

### CODE REVIEW

File: src/main/java/com/tropicalbastos/boids/settings/Platform.java [hunk 1]
```diff
@@ -9,11 +9,7 @@ public static String getOsName() {
...
    }
    
-   public static boolean isWindows() {
-      return getOsName().startsWith("Windows");
-   }
-
    public static boolean isMac() {
-      return getOsName().startsWith("mac");
-   }
+		return (getOsName().toLowerCase().indexOf("mac") >= 0);
```
Comment (TropicalBastos) [inline src/main/java/com/tropicalbastos/boids/settings/Platform.java]: Indentation a little steep here

**Review by TropicalBastos — COMMENTED** [review]
Just one minor thing, still need to test on my Mac

File: src/main/java/com/tropicalbastos/boids/settings/Platform.java [hunk 2]
```diff
@@ -9,11 +9,7 @@ public static String getOsName() {
       return OS;
    }
    
-   public static boolean isWindows() {
```
Comment (TropicalBastos) [inline src/main/java/com/tropicalbastos/boids/settings/Platform.java:12]: Also, no need to remove this, perhaps we'll need it later?

Comment (samkeshmiri) [reply src/main/java/com/tropicalbastos/boids/settings/Platform.java:12]: Good point, I'll add that in tomorrow, gotta catch some z's before work!

**Comment by TropicalBastos** [comment]
Great work @skeshmiri , merging!

**Pull request merged**